src/main/resources/com/google/jenkins/plugins/k8sengine/KubernetesEngineBuilder/config.jelly

```diff
@@ -29,10 +29,16 @@
     <f:entry field="manifestPattern" title="${%Kubernetes Manifests}">
         <f:textbox/>
     </f:entry>
-    <!-- TODO(craigatgoogle): re-enable once these QOL features are complete.
-    <f:entry field="verifyDeployments" title="${%Verify Deployments}">
-        <f:checkbox name="verifyDeployments" checked="${it.verifyDeployments}"/>
-    </f:entry>
+    <f:block>
+      <table>
+        <f:optionalBlock field="verifyDeployments" title="Verify Deployments" checked="${it.verifyDeployments}" inline="true">
+          <f:entry field="verifyTimeoutInMinutes" title="${%Verify Timeout in Minutes}">
+            <f:textbox type="number" default="5"/>
+          </f:entry>
+        </f:optionalBlock>
+      </table>
+    </f:block>
+    <!-- TODO
```

Comment:
This TODO should have an owner and brief description like the one deleted above. Since you're working on the service verification too that would be `TODO(johnlabarge): ...`

Reply:
will do thanks!

Reply:
Note that verifying services may require verifying deployments anyway.  I'll nest the TODO there for now.  We can talk offline.